classes/class-wc-connect-nux.php

```diff
@@ -60,6 +64,55 @@ public function check_notice_dismissal() {
 			$notices[ $notice ] = true;
 			update_option( 'wc_connect_nux_notices', $notices );
 		}
-	}
 
-}
\ No newline at end of file
+		private function init_pointers() {
+			add_filter( 'wc_services_pointer_woocommerce_page_wc-settings', array( $this, 'register_add_service_to_zone_pointer' ) );
+		}
+
+		public function show_pointers( $hook ) {
+			$pointers = apply_filters( 'wc_services_pointer_' . $hook, array() );
+
+			if ( ! $pointers || ! is_array( $pointers ) ) {
+				return;
+			}
+
+			$dismissed_pointers = explode( ',', (string) get_user_meta( get_current_user_id(), 'dismissed_wp_pointers', true ) );
+			$valid_pointers = array();
+
+			if( isset( $dismissed_pointers ) ) {
+				foreach ( $pointers as $pointer ) {
+					if ( ! in_array( $pointer['id'], $dismissed_pointers ) ) {
+						$valid_pointers[] =  $pointer;
+					}
+				}
+			} else {
+				$valid_pointers = $pointers;
+			}
+
+			if ( empty( $valid_pointers ) ) {
+				return;
+			}
+
+			wp_enqueue_style( 'wp-pointer' );
+			$wc_connect_base_url = defined( 'WOOCOMMERCE_CONNECT_DEV_SERVER_URL' ) ? WOOCOMMERCE_CONNECT_DEV_SERVER_URL : plugins_url( 'dist/', __FILE__ );
+			wp_register_script( 'wc_services_admin_pointers', $wc_connect_base_url . 'woocommerce-services-admin-pointers.js', array( 'wp-pointer', 'jquery' ), false, true );
```

Comment:
Nice! I made the change in 30f64a7ae69556d1bcc73fce429f3c08fc6b142c, also moving wc_connect_base_url to a class property. I defined in in `__construct()` - let me know if there's a better place, though.